.ado/windows-vs-pr.yml

```diff
@@ -104,12 +104,11 @@ jobs:
         condition: and(succeeded(), not(startsWith(variables.BuildPlatform, 'arm')))
 
       - task: VSTest@2
-        displayName: Run Universal Unit Tests (Managed)
+        displayName: Run Universal Unit Tests (UAP)
```

Comment:
UWP?

Reply:
I used the TFM name's. I think we (msft) need to work on getting more consistent branding for tech and not have many names for the same thing :)

Reply:
@asklar: Do you know why code coverage doesn't work for this? The NetCore tests is properly uploading coverage. I can understand for native without instrumenting the binaries, but since this framework is managed, I'd expect code coverage to work....